apps/profiles/migrations/0003_privacy_visible_as_attending_events.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.8 on 2018-02-04 20:41
```

Comment:
Delete this

Reply:
Why delete this? It's required to migrate the database from one state to another.